src/styles/block/calendar.scss

```
@@ -0,0 +1,33 @@
+.calendar {
+  display: flex;
+  max-width: $calendar-size;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  margin: 0 auto;
+  padding: 100px 0;
+
+  &__days {
+    transition: color 0.5s, transform 0.5s;
+    position: relative;
+
+    background-color: #eee;
+    border: 1px solid black;
```

don't forget to create variables
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
    position: relative;

    background-color: $c-color;
    border: 1px solid $c-black;
```